demo/full/scripts/controllers/ContentList.jsx

```diff
@@ -55,6 +55,9 @@ class ContentList extends React.Component {
       url,
       transport: this.state.techno.toLowerCase(),
       autoPlay: true, // TODO add checkBox
+      // native browser subtitles engine (VTTCue) doesn't render stylized subs
+      // we force HTML textTrackMode to vizualise styles
+      textTrackMode: "html",
```

Comment:
Why not always setting textTrackMode to ``"html"`` by default?

That is, not only for ``loadUrl`` but also ``loadContent``?

The only left problem I see with that is that going into fullscreen mode won't display text tracks. We could very easily fix this issue but we would be left with needing a progress bar for when the content is fullScreen.

Reply:
As ``loadContent`` takes as argument options specific to contents, maybe we shouldn't force this kind of options to a specific value. We may need to display subtitles in a native way for some contents.